### ISSUE TITLE
Breast granuloma

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10127]
**Preferred term label:**
Breast granuloma

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Granulomatous lobular mastitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000635
- label: Breast granuloma
- parents: [Granuloma (HP:0032252)](https://hpo.jax.org/app/browse/term/HP:0032252)
- pmid: PMID:30800035
- def: Formation of a non-necrotizing granuloma in breat tissue typically in combination with a localized infiltrate of multi-nucleated giant cells, epithelioid histiocytes, lymphocytes, and plasma cells.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10127
- synonyms: Granulomatous mastitis(NONE)
